### PR TITLE
New parameter to force open-vm-tools package

### PR DESCRIPTION
**Pull request by anders-larsson**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -55,28 +56,39 @@
...
-      'SLED', 'SLES': {
-        $_use_open_vm_tools = $lsbmajdistrelease_int >= 12
-      }
-      'OpenSuSE': {
-        $_use_open_vm_tools = $lsbmajdistrelease_int >= 12
-      }
-      'Ubuntu': {
-        if $prefer_open_vm_tools_real == true {
-          # include Ubuntu 12.04
+    if is_string($force_open_vm_tools) == true {
```
Comment (emahags) [inline manifests/init.pp]: @anders-larsson I'd like some spec tests for this as well (both true and 'true' should do the same thing)

Comment (Phil-Friderici) [inline manifests/init.pp]: Yeah ! Take care for stringified variable types. For testing I recommend to use the newest incarnation of my variable type validations https://github.com/ghoneycutt/puppet-module-ssh/blob/v3.34.0/spec/classes/init_spec.rb#L3482-L3511

Comment (emahags) [inline manifests/init.pp]: @Phil-Friderici Nice! @anders-larsson do it like that!

**Comment by emahags** [comment]
Thanks for your contribution!

**Pull request merged**